views/components/BarGraphTooltip/index.jsx

```diff
@@ -13,26 +13,32 @@ const BarGraphTooltip = ({
 }) => {
   if (!active || !payload?.length) return null
 
+  const siteName = useSiteName ? label : SITE_NAMES[label]
+  const countsTotal = useSiteName
+    ? studyTotals[label].count
+    : studyTotals[SITE_NAMES[label]].count
+  const variableCounts = payload[0].payload.counts
+
   return (
     <div className="BarGraphTooltip">
       {displayLeft ? <div className="arrow-left" /> : null}
       <div className="BarGraphTooltip_data">
-        <div className="BarGraphTooltip_label">{label}</div>
+        <div className="BarGraphTooltip_label">{siteName}</div>
         <div className="BarGraphTooltip_values">Value</div>
-        {payload.map(({ name, value }) => {
+        <span className="BarGraphTooltip_divider" />
```

Comment:
```suggestion
        <div className="BarGraphTooltip_divider" />
```

Reply:
I'll leave the span

Reply:
I'm not going to die on this hill but I'm curious why you chose the span in the first place. A div feels like the right choice here, vs a span which is usually used in text nodes.

Reply:
I considered using a div, and a p tag, but a span felt right. To make an underline just need a generic html component, if data was being presented and i need a block level element then a div is the right choice.